css/about.css

```diff
@@ -0,0 +1,202 @@
+@import url(./home.css);
+
+.navbar {
+  top: 0;
+  position: fixed;
+}
+
+.about-wrapper {
+  position: relative;
+  top: 200px;
+  width: 100%;
+}
+
+.about-us,
+.contact {
+  width: 60%;
+  margin: 10px auto;
+}
+
+.about-quote {
+  display: inline-block;
+  color: #ec5242;
+  font-family: 'cocogoose1', sans-serif;
+  font-size: 1rem;
+  padding-bottom: 5px;
+}
+
+.about-goal {
+  margin-top: 10px;
+  color: #ec5242;
+  font-family: 'cocogoose', sans-serif;
+  text-transform: uppercase;
+}
+
+.about-ideal {
+  line-height: 1.5rem;
+  padding: 10px;
+  margin: 20px auto;
+  width: 70%;
+  border: 3px solid #d3d3d3;
+  margin-bottom: 30px;
+  font-family: 'cocogoose1', sans-serif;
+  background-color: #fff;
+  box-shadow: 5px 5px 5px #d3d3d3, -5px -5px 5px #d3d3d3;
+}
+
+.about-logo {
+  background-color: #fff;
+  border: 1px solid #d3d3d3;
+  margin-top: 80px;
+  text-align: center;
+  height: 300px;
+}
+
+.title {
+  padding: 40px 0 15px;
+}
+
+.title::after {
+  content: '';
+  position: absolute;
+  width: 10%;
+  background-color: #ec5242;
+  transform: translate(-50%, -50%);
+  left: 48%;
+  height: 1px;
+  margin: 5px 25px;
+}
+
+.concern p {
+  font-family: 'cocogoose1', sans-serif;
+  font-size: 0.8rem;
+  width: 90%;
+  margin: 20px auto;
+  text-align: justify;
+}
+
+.past-activities {
+  text-align: center;
+  margin: 40px 0;
+}
+
+.past-activities p {
+  font-family: 'cocogoose1', sans-serif;
+  font-size: 0.8rem;
+  padding: 20px 0;
+}
+
+.activities-details {
+  width: 50%;
+  margin: 50px auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.activity1 {
+  position: relative;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 300px;
+  color: #fff;
+  height: 250px;
+  background: linear-gradient(#ec524280, #ec524280), url('../images/feature-image003.jpg') center/cover no-repeat;
+}
+
+.activity2 {
+  position: relative;
+  text-align: center;
+  width: 300px;
+  color: #fff;
+  height: 250px;
+  background: linear-gradient(#ec524280, #ec524280), url('../images/walk-the-talk-2018.jpg') center/cover no-repeat;
+}
+
+.activity1:hover,
+.activity2:hover {
+  border: 3px solid #ec5242;
+  cursor: pointer;
+}
+
+.activity1 div,
+.activity2 div {
+  position: absolute;
+  transform: translate(-50%, -50%);
+  left: 50%;
+  top: 50%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: 'cocogoose1', sans-serif;
+}
+
+.date {
+  font-size: 1.4rem;
+  font-family: 'cocogoose', sans-serif;
+}
+
+.about-last {
+  background-color: #000;
+  color: #fff;
+  height: 50px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  padding: 30px;
+}
+
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] **Required:** [When building a mobile-first website you use `min-width`](https://www.mightyminnow.com/2013/11/what-is-mobile-first-css-and-why-does-it-rock/) queries as you have rightly done in the `home.css` file. Kindly target mobile-first.

![image](https://user-images.githubusercontent.com/27709832/163600411-dc5a7e68-bd37-49da-bf62-4665541b3fb1.png)

Reply:
I have forgoten here! the fact that I just finish to make the home page desktop version so I thought I was still working it. 
:pleading_face:

Reply:
resolved 👍